exe/Wrapper.hs

```diff
@@ -26,16 +26,28 @@ main :: IO ()
 main = do
   -- WARNING: If you write to stdout before runLanguageServer
   --          then the language server will not work
-  Arguments{..} <- getArguments "haskell-language-server-wrapper"
+  args <- getArguments "haskell-language-server"
```

Comment:
Just a question: why do you change the name for the wrapper in help/version message?

Reply:
Very good question!
Copy and paste error, thank you for catching it!